components/event-publisher-proxy/README.md

```diff
@@ -31,6 +31,7 @@ $ ko apply -f config/
 
 ### Send Events
 
+- Supports cloud events
```

Comment:
```suggestion
This command supports **cloud events**: 
```

I think there should be some kind of full sentence here so it is easier to read for the user.